MobileCenter/MobileCenter/Internals/Sender/MSHttpSender.h

```diff
@@ -23,7 +22,12 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  *  Pending http calls.
  */
-@property(atomic, strong) NSMutableDictionary<NSString *, id <MSSenderCall>> *pendingCalls;
+@property(atomic, strong) NSMutableDictionary<NSString *, MSSenderCall *> *pendingCalls;
+
+// TODO (jaelim): Add doc here.
+- (void)sendAsync:(NSObject *)container
```

Comment:
I tried but it looks I couldn't. `NSSender` is a protocol that can't have generics. Maybe I can try with `__kindof` but I'm not sure how it will work at the moment.
I'd like to leave this as it is for now, we can use generics later.